src/Validation/Validation.php

```diff
@@ -306,14 +306,34 @@ public static function comparison($check1, $operator, $check2)
      * @param string $field The field to check $check against. This field must be present in $context.
      * @param array $context The validation context.
      * @return bool
+     * @deprecated 3.6.0 Use compareFields() instead.
```

Comment:
Not sure this has to be deprecated. The new function is slightly more awkward to use because of the additional arguments.